cime/src/drivers/mct/main/prep_glc_mod.F90

```diff
@@ -1108,4 +1443,176 @@ function prep_glc_get_mapper_Fl2g()
     prep_glc_get_mapper_Fl2g => mapper_Fl2g
   end function prep_glc_get_mapper_Fl2g
 
+  function prep_glc_get_mapper_So2g()
+    type(seq_map), pointer :: prep_glc_get_mapper_So2g
+    prep_glc_get_mapper_So2g=> mapper_So2g
+  end function prep_glc_get_mapper_So2g
+
+  function prep_glc_get_mapper_Fo2g()
+    type(seq_map), pointer :: prep_glc_get_mapper_Fo2g
+    prep_glc_get_mapper_Fo2g=> mapper_Fo2g
+  end function prep_glc_get_mapper_Fo2g
+
+!***********************************************************************
+!
+!  routine compute_melt_fluxes
+!
+!> \brief   Computes ocean and ice melt fluxes, etc.
+!> \author  Xylar Asay-Davis
+!> \date    3/27/2015
+!>  This routine computes melt fluxes (melt rate, temperature fluxes
+!>  into the ice and the ocean, and salt flux) as well as the interface
+!>  temperature and salinity.  This routine expects an ice temperature
+!>  in the bottom layer of ice and ocean temperature and salinity in
+!>  the top ocean layer as well as the pressure at the ice/ocean interface.
+!>
+!>  The ocean heat and salt transfer velocities are determined based on
+!>  observations of turbulent mixing rates in the under-ice boundary layer.
+!>  They should be the product of the friction velocity and a (possibly
+!>  spatially variable) non-dimenional transfer coefficient.
+!>
+!>  The iceTemperatureDistance is the distance between the location
+!>  where the iceTemperature is supplied and the ice-ocean interface,
+!>  used to compute a temperature gradient.  The ice thermal conductivity,
+!>  SHR_CONST_KAPPA_LAND_ICE, is zero for the freezing solution from Holland and Jenkins
+!>  (1999) in which the ice is purely insulating.
+!
+!-----------------------------------------------------------------------
+
+  subroutine compute_melt_fluxes( &
```

Comment:
This routine does not match the style used throughout cime's Fortran code, and should be cleaned up. (Granted, the existing code style isn't fully consistent, but this routine differs more substantially than normal.):

- The above documentation should be edited to match more typical documentation styles in cime

- Whitespace should be cleaned up to match typical indentation (e.g., indentation of continuation lines for the argument list)

- Variable declaration lines should be edited to match the common style: `real(r8)` rather than `real (kind=r8)`, separate declaration lines for each variable, `real(r8) :: foo(:)` rather than using the `dimension` keyword, etc.

Reply:
@billsacks , I find this routine to be easier to read than most of CIME, so I'd prefer to keep it as stylistically written, even if it differs from other routines, particularly given it's a specialized routine that is not at all central to the workings of the module.  But if you feel particularly strongly about this, let me know.